docs/conf/eu/2017/news/index.rst

```diff
@@ -10,6 +10,7 @@ More here as it happens!
 .. toctree::
    :glob:
 
+   announcing-events-volunteers
```

Comment:
@thatdocslady sadly we gotta do this, if you just glob the files like it was before, the order doesn't work.